ush/ICpert_scripts/get_gefs_rrfs.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash --login
+#SBATCH --account=fv3lam
+#SBATCH --partition=service 
+#SBATCH --job-name=ens_gefs
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --time=5:00:00
+
+#for 30h gefs perturbations to a 00z experiment init, use the 12z init forecasts from 1 day before
+
+set -x
+
```

Comment:
I see some scripts have "set -x" and others don't.  Is that done on purpose?

Reply:
No.  This is not one of the scripts I worked on.  I think @briannen committed this last week, and she chose to include this to get more debugging info.

Reply:
Got it.  Thanks!